client.go

```diff
@@ -311,6 +311,10 @@ func NewClientFromInfo(info ConnectInfo) (*Client, error) {
 	c := &Client{
 		// Config: *config,
 		Http: http.Client{},
+		Config: Config{
+			Remotes: DefaultRemotes,
+			Aliases: map[string]string{},
```

Comment:
Because it's just copied from DefaultConfig :). I suppose I could use DefaultConfig directly, but I didn't know what the implications of setting a default remote are.